submissions/evgenii-del/memory-game/js/main.js

```diff
@@ -0,0 +1,121 @@
+const cards = [
+    {
+        id: 1,
+        img: 'images/card1.jfif'
+    },
+    {
+        id: 2,
+        img: 'images/card2.jfif'
+    },
+    {
+        id: 3,
+        img: 'images/card3.jfif'
+    },
+    {
+        id: 4,
+        img: 'images/card4.jfif'
+    },
+    {
+        id: 5,
+        img: 'images/card5.jfif'
+    },
+    {
+        id: 6,
+        img: 'images/card6.jfif'
+    },
+    {
+        id: 7,
+        img: 'images/card7.jfif'
+    },
+    {
+        id: 8,
+        img: 'images/card8.jfif'
+    }
+];
+const cards_container = document.querySelector('.js-cards_container');
+const finalStep = 8;
+let hasFlipped, playing = false;
+let firstCard, secondCard, curStep;
+
+const createItem = (card) => {
+    const block = document.createElement('div');
+    block.classList.add('flip-container');
+    block.innerHTML = `
+            <div class="flipper card" data-id="${card.id}">
+                <div class="front">
+                    <img src="images/card-front.jfif" alt="card-front">
+                </div>
+                <div class="back">
+                    <img src="${card.img}" alt="${card.img}">
+                </div>
+            </div>
+        `;
+    return block;
+}
+
+const renderItems = (cards) => {
+    let fragment = document.createDocumentFragment();
```

Comment:
It's not a blocker, but it's better to use `const` for the variables which don't need reassignment.